From d3dc207229be0d9df878c2a9f929b22a3ebfb4b5 Mon Sep 17 00:00:00 2001
From: Ganesh Kariganuru Mahabalesh <quic_gkarigan@quicinc.com>
Date: Wed, 10 Jan 2024 15:04:54 -0800
Subject: [PATCH] wifi: ath12k: Update the DS pMSI IRQs naming convention.

Currently, In DS both pMSI and MSI IRQs are having the convention
"pci%d_xxxxxx_%d" where initial format specifier represents bus_id
and the later ppeds_node_id.

Update the convention to "pcic%d_xxxxx_%d" in case of DS  pMSI IRQs

Signed-off-by: Ganesh Kariganuru Mahabalesh <quic_gkarigan@quicinc.com>

diff --git a/drivers/net/wireless/ath/ath12k/pcic.c b/drivers/net/wireless/ath/ath12k/pcic.c
index a8b3eee..04009f5 100644
--- a/drivers/net/wireless/ath/ath12k/pcic.c
+++ b/drivers/net/wireless/ath/ath12k/pcic.c
@@ -607,7 +607,7 @@ int ath12k_pcic_ppeds_register_interrupts(struct ath12k_base *ab, int type, int
 			goto irq_fail;
 		irq_set_status_flags(irq, IRQ_DISABLE_UNLAZY);
 		snprintf(&ab->dp.ppeds_irq_name[PPEDS_IRQ_PPE2TCL], sizeof(ab->dp.ppeds_irq_name),
-			 "pci%d_ppe2tcl_%d", bus_id, ab->ppeds_node_idx);
+			 "pcic%d_ppe2tcl_%d", bus_id, ab->ppeds_node_idx);
 		ret = devm_request_irq(&pdev->dev, irq,  ath12k_ds_ppe2tcl_irq_handler,
 				  IRQF_SHARED,
 			    ab->dp.ppeds_irq_name[PPEDS_IRQ_PPE2TCL], (void *)ath12k_dp_get_ppe_ds_ctxt(ab));
@@ -620,7 +620,7 @@ int ath12k_pcic_ppeds_register_interrupts(struct ath12k_base *ab, int type, int
 			goto irq_fail;
 		irq_set_status_flags(irq, IRQ_DISABLE_UNLAZY);
 		snprintf(&ab->dp.ppeds_irq_name[PPEDS_IRQ_REO2PPE], sizeof(ab->dp.ppeds_irq_name),
-			 "pci%d_reo2ppe_%d", bus_id, ab->ppeds_node_idx);
+			 "pcic%d_reo2ppe_%d", bus_id, ab->ppeds_node_idx);
 		ret = devm_request_irq(&pdev->dev, irq,  ath12k_ds_reo2ppe_irq_handler,
 				  IRQF_SHARED,
 				  ab->dp.ppeds_irq_name[PPEDS_IRQ_REO2PPE], (void *)ath12k_dp_get_ppe_ds_ctxt(ab));
@@ -633,7 +633,7 @@ int ath12k_pcic_ppeds_register_interrupts(struct ath12k_base *ab, int type, int
 			goto irq_fail;
 		irq_set_status_flags(irq, IRQ_DISABLE_UNLAZY);
 		snprintf(&ab->dp.ppeds_irq_name[PPEDS_IRQ_PPE_WBM2SW_REL], sizeof(ab->dp.ppeds_irq_name),
-			 "pci%d_ppe_wbm_rel_%d", bus_id, ab->ppeds_node_idx);
+			 "pcic%d_ppe_wbm_rel_%d", bus_id, ab->ppeds_node_idx);
 		ret = devm_request_irq(&pdev->dev, irq,  ath12k_dp_ppeds_handle_tx_comp,
 				  IRQF_SHARED,
 				  ab->dp.ppeds_irq_name[PPEDS_IRQ_PPE_WBM2SW_REL],(void *)ab);
-- 
2.25.1

